### DISCUSSION TITLE
Submit our patches to the upstream

### DISCUSSION THREAD

**Discussion by gtxzsxxk** [gtxzsxxk/duo-buildroot#17] [General]
I prepare to do this after 18th Nov when I will go through my mid-term.

**Comment by shodan8192** [comment]
OK, in the meantime I'll continue to work. I don't want to distract you, should I PR if I make something ?

**Comment by gtxzsxxk** [comment]
Yes, please. Feel free to PR and I will handle your PRs when I am available.
